bedrock/mozorg/templates/mozorg/home/home.html

```diff
@@ -233,10 +243,17 @@ <h2 class="primary go">{{ _('Teach the Web') }}</h2>
           </a>
         </li>
       {% endif %}
-      {% if l10n_has_tag('gear_store') %}
+      {# only show webmaker promo below if new gear store promo is shown #}
+      {% if l10n_has_tag('gear_promo_eoy') %}
```

Comment:
This doesn't guarantee that a locale which has the `gear_promo_eoy` tag also has the web maker strings translated. Perhaps we could do something like `if not l10n_has_tag('gear_promo_eoy') and l10n_has_tag('webmaker_promo_introducing')`?

Does this make sense @flodolo?